.travis.yml

```diff
@@ -13,11 +13,14 @@ matrix:
   include:
     - php: 7.1
       env: PHPSTAN=1
+           CODING_STANDARD=1
 
 before_script:
   - composer --prefer-source install
   - if [[ $PHPSTAN = 1 ]]; then composer require --dev phpstan/phpstan:^0.7; fi
+  - if [[ $CODING_STANDARD = 1 ]]; then composer require --dev symplify/easy-coding-standard:^2.2; fi
```

Comment:
I'm -1 on adding an unnecessary dependency on an _unknown_ project with questionable stability (some of us remember how @TomasVotruba's repositories magically disappeared without any deprecation/abandonment/notice from GitHub, breaking any and all depending libraries/code :/). Additionally it brings barely any added value over just using PHPCS directly (and a single custom rule would be stored in doctrine/coding-standard regardless so it wouldn't clutter other repositories). _</biased-opinion-based-on-experience>_